lib/ancestry/class_methods.rb

```diff
@@ -29,11 +35,26 @@ def scope_depth(depth_options, depth)
     # To order your hashes pass the order to the arrange method instead of to the scope
 
     # Get all nodes and sort them into an empty hash
+    # Supports eager loading of associations
     def arrange(options = {})
+      scope = ancestry_base_class
+      
+      if (includes_param = options.delete(:includes))
+        scope = scope.includes(includes_param)
+      end
+      
+      if (preload_param = options.delete(:preload))
+        scope = scope.preload(preload_param)
+      end
+      
+      if (eager_load_param = options.delete(:eager_load))
```

Comment:
not sure if we want to pull off of `arrange` or we want to pull off the scope coming in.

E.g., `targets.try(:includes_value)` (thought we'd probably use standard and not `try`)